src/comps/qb.cc

```diff
@@ -1,6 +1,7 @@
-#include <RandBLAS.hh>
-#include <lapack.hh>
-#include <RandLAPACK.hh>
+#include "RandLAPACK.hh"
```

Comment:
Following my comment on ``src/comps/orth.cc``: I think this include statement of RandLAPACK can be changed to ``#include "RandLAPACK/comps/qb.hh"``.

Reply:
OK, I'll give it  a try

Reply:
done